project-service/api.md

```diff
@@ -0,0 +1,594 @@
+# Project Service External API
+
+## Users
+
+### Fetch Users
+
+Returns json data of all users as an array
+
+- URL
+  + `/users`
```

Comment:
when would returning all users be useful?

returning all members of a specific project (`/projects/:project_id/users`) or of a specific organization (`/organizations/:organization_id/users`), on the other hand...